### ISSUE TITLE
Check if secure ciphers aren't used in CloudFront

### ISSUE THREAD

**Issue by frediana** [Checkmarx/kics#3692]
### Description

I have a terraform code, creating an AWS Cloudfront resource, configured as this:

```
resource "aws_cloudfront_distribution" "foo" {
...
  viewer_certificate {
    cloudfront_default_certificate = false
    acm_certificate_arn            = var.my_certificate
    minimum_protocol_version       = "TLSv1.2_2019"
    ssl_support_method             = "sni-only"
  }
...
}
```

### What I Did

Checkmarks kiks is not happy:

```
Secure Ciphers Disabled, Severity: HIGH, Results: 1
Description: Check if secure ciphers aren't used in CloudFront
Platform: Terraform
	[1]: /builds/AykYiFxz/3/butterfly/butterfly/infra/terraform/front/cdn.tf:73
		072:     acm_certificate_arn            = var.shared.certificates.north_virginia.arn
		073:     minimum_protocol_version       = "TLSv1.2_2019"
		074:     ssl_support_method             = "sni-only"
```

I think this should not be raised.

By reading the definition of this rule here: https://github.com/Checkmarx/kics/blob/3d8a6cc4268827ed837e0f84fb8b84aa3f1e7a01/assets/queries/terraform/aws/secure_ciphers_disabled/query.rego

I can see it's expected to have minimum_protocol_version  strictly equal to TLSv1.2 or TLSv1.1. 

To me this should be more something like minimum_protocol_version  should **start with**  TLSv1.2 or TLSv1.1.

See https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/secure-connections-supported-viewer-protocols-ciphers.html#secure-connections-supported-ciphers for the supported list.

If the issue is confirmed, I would be happy to contribute.

Adrien

**Comment by felipe-avelar** [comment]
Hi, @frediana, I hope you're doing well.
After analyzing this issue, we agree with you, that we should check with its `starts_with` these protocols and we will be very happy if you could give a contribution in this.

**Comment by frediana** [comment]
Ok, no problem a PR should come soon then. Thank you

**Comment by rogeriopeixotocx** [comment]
Yes I believe we can check with OPA built-in function `startswith(string, 'TLSv1.2')` https://www.openpolicyagent.org/docs/latest/policy-reference/#strings
